File: VB/LayoutControlItemsSource/ItemTemplateSelector.vb
Imports System.Windows
Imports System.Windows.Controls

Namespace DXSample
    Public Class ItemTemplateSelector
        Inherits DataTemplateSelector

        Public Property DefaultTemplate() As DataTemplate
        Public Overrides Function SelectTemplate(ByVal item As Object, ByVal container As System.Windows.DependencyObject) As DataTemplate
            Return DefaultTemplate
        End Function
    End Class
End Namespace

File: CS/LayoutControlItemsSource/ItemTemplateSelector.cs
using System.Windows;
using System.Windows.Controls;

namespace DXSample {
    public class ItemTemplateSelector : DataTemplateSelector {
        public DataTemplate DefaultTemplate { get; set; }
        public override DataTemplate SelectTemplate(object item, System.Windows.DependencyObject container) {
            return DefaultTemplate;
        }
    }
}